src/ca65/instr.c

```diff
@@ -303,6 +303,97 @@ static const struct {
     }
 };
 
+/* Instruction table for the 6502 with illegal instructions (X) and DTV 
+** extra opcodes (DTV). Some illegal instructions (X, -DTV?) might be not 
+** supported by DTV. They should be tested on the DTV hardware. 
+*/
+static const struct {
+    unsigned Count;
+    InsDesc  Ins[78];
+} InsTab6502DTV = {
+    sizeof (InsTab6502DTV.Ins) / sizeof (InsTab6502DTV.Ins[0]),
+    {
+        { "ADC",  0x080A26C, 0x60, 0, PutAll },
+        { "ALR",  0x0800000, 0x4B, 0, PutAll },         /* X */
+        { "ANC",  0x0800000, 0x0B, 0, PutAll },         /* X -DTV? */
```

Comment:
I guess the comment wants  to read `X, -DTV` (with comma) here too...

Reply:
You are right, I have corrected the referred mistyped comment.